packages/simple-schema/lib/content-assist/element-name.js

```diff
@@ -1,5 +1,7 @@
-const { difference, map, filter } = require("lodash");
+const { difference, map, filter, find } = require("lodash");
 
+// https://www.w3.org/TR/2009/REC-xml-names-20091208/#NT-PrefixedName
+const NAMESPACE_PATTERN = /^(?:([^:]*):)?([^:]*)$/;

```

Comment:
evaluate using **named capturing groups**
it is available in node.js >= 10
And node.js 8 will be deprecated in end of this month